frontend-svelte/rollup.config.js

```diff
@@ -40,13 +40,14 @@ export default {
 		file: 'public/build/bundle.js'
 	},
 	plugins: [
+	
 		json(),
 		replace({
-			process: JSON.stringify({
-				env: {
-					isProd: production,
-				}
-			}),
+			// process: JSON.stringify({
+			// 	env: {
+			// 		isProd: production,
+			// 	}
+			// }),
```

Comment:
This was replacing `process` including functions called `process` in any scope. Do we need to replace the whole process object? Where is `process.env.isProd` used? I couldn't find it when i had a quick look.